tests/cypress/integration/reportRepair/smellGasMvp.spec.js

```diff
@@ -0,0 +1,14 @@
+describe('smellGas', () => {
+  context('conditional mvp tests', () => {
+    if (Cypress.env('CYPRESS_RELEASE_VERSION') == 'mvp') {
+      beforeEach(() => {
+        cy.visit('http://localhost:3000/report-repair/smell-gas');
+      });
+
+      it('Displays mvp title', () => {
```

Comment:
This contains a couple of validations which would be best to split into their own tests.
The current way it's written won't attempt the second check if the first fails, which _could_ hide/not show a relevant fail.
In line with the above, the title of this test doesn't reflect both of the things it checks.